ts/src/pro/gate.ts

```diff
@@ -345,6 +326,24 @@ export default class gate extends gateRest {
         //          "low_24h": "42721.03"
         //        }
         //    }
+        //
+        this.helperForHandleTickerBidAsk ('ticker', client, message);
+    }
+
+    async watchBidsAsks (symbols: Strings = undefined, params = {}): Promise<Tickers> {
+        /**
+         * @method
+         * @name gate#watchBidsAsks
+         * @description watches bids & asks for symbols
```

Comment:
can you add the docs here?

Reply:
binance - https://binance-docs.github.io/apidocs/futures/en/#all-book-tickers-stream
gate
gemini
... would be more, if needed i'll research further, just those three above are the ones i've already looked into